multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -318,81 +314,6 @@ const toAmt = (
   balance: QueryBalanceResponseSDKType['balance'],
 ) => make(brand, BigInt(balance?.amount || 0));
 
-test.serial('lp deposits', async t => {
-  const { lpUser, retryUntilCondition, vstorageClient, wallets } = t.context;
-
-  const lpDoOffer = makeDoOffer(lpUser);
-
-  const { USDC, FastLP } = await agoricNamesQ(vstorageClient).brands('nat');
-
-  const give = { USDC: make(USDC, LP_DEPOSIT_AMOUNT) };
-
-  const metricsPre = await fastLPQ(vstorageClient).metrics();
-  const want = { PoolShare: divideBy(give.USDC, metricsPre.shareWorth) };
-
-  const proposal: USDCProposalShapes['deposit'] = harden({ give, want });
-  await lpDoOffer({
-    id: `lp-deposit-${Date.now()}`,
-    invitationSpec: {
-      source: 'agoricContract',
-      instancePath: [contractName],
-      callPipe: [['makeDepositInvitation']],
-    },
-    proposal,
-  });
-
-  await retryUntilCondition(
-    () => fastLPQ(vstorageClient).metrics(),
-    ({ shareWorth }) =>
-      !isGTE(metricsPre.shareWorth.numerator, shareWorth.numerator),
-    'share worth numerator increases from deposit',
-    { log },
-  );
-
-  const { useChain } = t.context;
-  const queryClient = makeQueryClient(
-    await useChain('agoric').getRestEndpoint(),
-  );
-
-  await retryUntilCondition(
-    () => queryClient.queryBalance(wallets['lp'], 'ufastlp'),
-    ({ balance }) => isGTE(toAmt(FastLP, balance), want.PoolShare),
-    'lp has pool shares',
-    { log },
-  );
-  t.pass();
-});
-
-test.serial('reconfigure: fix noble ICA', async t => {
-  const { startContract, commonBuilderOpts } = t.context;
-  // TODO: use actual rc-1 artifact
-  const builder =
-    '../packages/fast-usdc-deploy/src/fast-usdc-reconfigure.build.js';
-
```

Comment:
Why is this now unneeded?

Reply:
I think @dckc put this here assuming the earlier eval would be with a released artifact but the multichain tests aren't meant to cover upgrades. The earlier eval is HEAD of the contract so doesn't need this subsequent reconfiguration

Reply:
> the multichain tests aren't meant to cover upgrades.

I think of it more as: we haven't managed to fit upgrades into multichain-testing, not that we don't want them there.

 - #10079

Reply:
That would just be a different baseline, not performing multiple evals in this test